app/views/shelters/show.html.erb

```diff
@@ -4,14 +4,16 @@
   <p><%= @shelter.city %>, <%= @shelter.state %> <%= @shelter.zip %></p>
   <br>
 
-  <section>
+  <div>
     <%= @shelter_reviews.each do |shelter_review| %>
       <p>Title: <%= shelter_review.title %></p>
-      <p>Rating: <%= shelter_review.rating %> old</p>
+      <p>Rating: <%= shelter_review.rating %></p>
       <p>Content: <%= shelter_review.content %></p>
-      <p>Image: <%= shelter_review.image %></p>
+      <p>Image: <img src="<%= shelter_review.image %>" style="width: 55vw; min-width: 330px;" ></p>
```

Comment:
This was me just playing around with styling when I hit a wall with the form. We don't have to keep this. I was just trying out ways to get the image to scale with the browser window size. It's not quite there yet as you can see. XD

Reply:
This is good to play around with.  We can definitely finish this together today.